installer/bundle/bundle_skel_AIX.sh

```diff
@@ -134,13 +192,52 @@ pkg_rm() {
 pkg_upd() {
     pkg_filename=$1
     pkg_name=$2
+    pkg_allowed=$3
 
     echo "----- Updating package: $pkg_name ($pkg_filename) -----"
 
+    if [ -z "${forceFlag}" -a -n "$pkg_allowed" ]; then
```

Comment:
Bug "-n" is `True if the length of "STRING" is non-zero.` Since `$pkg_allowed` is 0 or 1, it will always be true